src/Makefile

```diff
@@ -0,0 +1,200 @@
+# FV test Makefile
+
+all: stage-build
+
+ROOT := $(abspath $(shell pwd)/../)
+MK_DIR := ${ROOT}/mk
+
+GTEST_DIR := $(ROOT)/modules/gmock/gtest
+GMOCK_DIR := $(ROOT)/modules/gmock
+
+TARGET_TEST := cpp_common_fvtest
+
+TARGET_SOURCES_TEST := memcachedstore.cpp \
+                       memcachedstoreview.cpp \
+                       memcached_config.cpp \
+                       signalhandler.cpp \
+                       log.cpp \
+                       logger.cpp \
+                       fakelogger.cpp \
+                       test_main.cpp \
+                       test_memcachedstore.cpp \
+                       test_memcached_config.cpp
+
+TARGET_EXTRA_OBJS_TEST := gmock-all.o \
+                          gtest-all.o
+
+TEST_XML = $(TEST_OUT_DIR)/test_detail_$(TARGET_TEST).xml
+VG_XML = $(TEST_OUT_DIR)/vg_$(TARGET_TEST).memcheck
+VG_OUT = $(TEST_OUT_DIR)/vg_$(TARGET_TEST).txt
+VG_LIST = $(TEST_OUT_DIR)/vg_$(TARGET_TEST)_list
+
+VG_SUPPRESS = $(TARGET_TEST).supp
+
+EXTRA_CLEANS += $(TEST_XML) \
+                $(COVERAGE_XML) \
+                $(VG_XML) $(VG_OUT) \
+                $(OBJ_DIR_TEST)/*.gcno \
+                $(OBJ_DIR_TEST)/*.gcda \
+                *.gcov
+
+CPPFLAGS += -Wno-write-strings \
+            -ggdb3 -std=c++0x
+CPPFLAGS += -I${ROOT}/include \
+            -I${ROOT}/usr/include \
+            -I${ROOT}/modules/cpp-common/include \
+            -I${ROOT}/modules/sas-client/include \
+            -I${ROOT}/modules/jsoncpp/include \
+            -I${ROOT}/modules/rapidjson/include
+
+# Add cpp-common/src as a VPATH to pull in required production code
+VPATH := ${ROOT}/modules/cpp-common/src
+
+# Test build:
+#
+# Turn on code coverage.
+# Disable optimization, for speed and coverage accuracy.
+# Allow testing of private and protected fields/methods.
+# Add the Google Mock / Google Test includes.
+CPPFLAGS_TEST  += -DUNIT_TEST \
+                  -fprofile-arcs -ftest-coverage \
+                  -O0 \
+                  -fno-access-control \
+                  -I$(GTEST_DIR)/include -I$(GMOCK_DIR)/include \
+                  -I${ROOT}/modules/cpp-common/test_utils
+
+LDFLAGS += -L${ROOT}/usr/lib
+LDFLAGS += -lzmq \
+           -levent_pthreads \
+           -lmemcached \
+           -levent \
+           -lsas \
+           -lcurl \
+           -lpthread \
+           -lcares \
+           -lboost_regex \
+           -lboost_date_time \
+           -lc \
+           -lrt \
+           -lboost_system \
+           -lboost_filesystem
```

Comment:
Do we really use all of this list?  Though I guess it doesn't matter too much if we over-specify...